test/navigation/NomineesNavigatorSpec.scala

```diff
@@ -630,22 +663,16 @@ class NomineesNavigatorSpec extends SpecBase {
         }
       }
 
-      "from the IsOrganisationNomineePayments Page" must {
+      "from the OrganisationAuthorisedPersonName page" must {
 
         "go to the SessionExpiredController page when user answer is empty" in {
-          navigator.nextPage(IsOrganisationNomineePaymentsPage, CheckMode, emptyUserAnswers) mustBe
+          navigator.nextPage(OrganisationAuthorisedPersonNamePage, CheckMode, emptyUserAnswers) mustBe
             routes.SessionExpiredController.onPageLoad()
         }
 
-        "go to the What are Organisation's bank account details page when selected yes and clicked continue" in {
-          navigator.nextPage(IsOrganisationNomineePaymentsPage, CheckMode,
-            emptyUserAnswers.set(IsOrganisationNomineePaymentsPage, true).success.value) mustBe
-            routes.DeadEndController.onPageLoad()//TODO when next page created
-        }
-
-        "go to the Adding Authorised person from organisation page when selected no and continue button is clicked" in {
-          navigator.nextPage(IsOrganisationNomineePaymentsPage, CheckMode,
-            emptyUserAnswers.set(IsOrganisationNomineePaymentsPage, false).success.value) mustBe
+        "go to the what is full authorieds person's date of birth page when continue button is clicked" in {
```

Comment:
typo

Reply:
in fact, this doesn't seem to be a typo - shouldn't that go back to the summary page, rather than to the DOB page?